dev/src/codewind/project/Project.ts

```diff
@@ -464,6 +466,77 @@ export default class Project implements vscode.QuickPickItem {
         });
     }
 
+    public async onLoadRunnerUpdate(event: {projectID: string, status: string, timestamp: string}): Promise<void> {
+        Log.d(`${this.name} load runner status changed to ${event.status}`, event);
+        switch (event.status) {
```

Comment:
It seems weird to have a switch statement with two cases that are really just the same.
Why not:
```ts
if (![ "hcdReady", "profilingReady" ].includes(event.status)) {
  Log.e("Bad status or whatever, not sure what this means");    // or Log.i
  return;
}
// process the profiling data here

Reply:
Load runner emits events for starting/stopping/cancelled etc and we're just listening for those that tell the extension that data is ready.

I agree, much cleaner this way.